### PR TITLE
+ qfs 0.9

### PR DESCRIPTION
**Pull request by ygrek**
CI build is expected to fail due to absent libqfs

### CODE REVIEW

File: packages/qfs/qfs.0.9/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/ahrefs/ocaml-qfs"
+dev-repo: "git://github.com/ahrefs/ocaml-qfs.git"
+bug-reports: "https://github.com/ahrefs/ocaml-qfs/issues"
+tags: [ "org:ahrefs" "clib:stdc" "clib:qfs"  ]
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"] {with-test}
+  ["ocaml" "setup.ml" "-build"] {with-test}
```
Comment (kit-ty-kate) [inline packages/qfs/qfs.0.9/opam]: the build and configure steps are going to be done twice. I don't think this is what you wanted to do. I would suggest to place the `{with-test}` attribute to the argument for the configure and to omit the second build command line as it is redundant with the first one which is done unconditionally

File: packages/qfs/qfs.0.9/opam [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  ["ocaml" "setup.ml" "-test"] {with-test}
+  ["ocaml" "setup.ml" "-doc"] {with-doc}
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
```
Comment (kit-ty-kate) [inline packages/qfs/qfs.0.9/opam]: the remove field can be omitted with opam2

**Comment by ygrek** [comment]
thanks, fixed

**Comment by kit-ty-kate** [comment]
> I would suggest to place the {with-test} attribute to the argument for the configure and to omit the second build command line as it is redundant with the first one which is done unconditionally

sorry, what I meant about this comment is my latest commit

**Comment by kit-ty-kate** [comment]
It should be ok now. Thanks!

**Comment by ygrek** [comment]
I didn't know one could do that, nice!

**Pull request merged**